substrate/client/transaction-pool/tests/fatp_common/mod.rs

```diff
@@ -229,9 +229,14 @@ macro_rules! assert_future_iterator {
 macro_rules! assert_watcher_stream {
 	($stream:ident, [$( $event:expr ),*]) => {{
 		let expected = vec![ $($event),*];
-		log::debug!(target:LOG_TARGET, "expected: {:#?} {}, block now:", expected, expected.len());
+		tracing::debug!(
+    		target: LOG_TARGET,
+    		?expected,
+    		expected_len = expected.len(),
+    		"block now"
+		);
```

Comment:
not sure if this is intended, but feels like some indentation is missing
```suggestion
		tracing::debug!(
    			 target: LOG_TARGET,
    		 	 ?expected,
    	 		 expected_len = expected.len(),
    			 "block now"
		);
```

Reply:
@iulianbarbu weird that github is not rendering this properly. it is actually indented, you can see it here
https://github.com/dharjeezy/polkadot-sdk/blob/dami/transaction-pool-logging/substrate/client/transaction-pool/tests/fatp_common/mod.rs#L232